cp3pt0-deployment/setup_tenant.sh

```diff
@@ -139,6 +144,10 @@ function pre_req() {
         error "Cert-manager is not found or having more than one\n"
     fi
 
+    if [[ $LICENSE_ACCEPT != 1 ]]; then
```

Comment:
```suggestion
    if [ "$LICENSE_ACCEPT" -ne 1 ]; then
```
simpler form of checking